README.md

```diff
@@ -14,14 +14,14 @@ This provides an `apko_build` resource that will build the provided `apko` confi
 # with additional packages.
 provider "apko" {
   # Default to building for these architectures.
-  archs = ["x86_64", "aarch64"]
+  default_archs = ["x86_64", "aarch64"]
```

Comment:
IIRC this overwrites a config's archs if set, but we could double-check that with a test.

Reply:
Hmm, yeah lemme check that.

Reply:
Ok, I updated this with the new semantic and a test for that.